frontend/src/app/features/assistance-selection/assistance-selection.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AssistanceLevel } from 'src/app/core/interfaces/assistance-level';
```

Comment:
Is the interface needed now?

Reply:
It shouldn't be I will remove it

Reply:
I removed the import and fixed 2 other minor things I found.